### PR TITLE
Dummy

### PR DESCRIPTION
**Pull request by dhruv-mahajan**
PR after incorporating review comments

### CODE REVIEW

File: src/main/java/dummyPack/FizzBuzz.java [hunk 1]
```diff
@@ -4,31 +4,31 @@
 
 public class FizzBuzz {
     public static void main(String[] args) {
-        Scanner scanner =new Scanner(System.in);
+        Scanner scanner = new Scanner(System.in);
```
Comment (shivangi8293) [inline src/main/java/dummyPack/FizzBuzz.java]: Main method is not required

**Comment by vishalsinha21** [comment]
good job 👍

**Comment by dhruv-mahajan** [comment]
Changes done and main method removed as well.

**Review by shivangi8293 — APPROVED** [review]
The changes are reviewed. Please proceed with merging.

**Pull request merged**